### PR TITLE
Add IPMI KCS function test

### PR DESCRIPTION
**Pull request by sharkconi**
Signed-off-by: Mark.Ma <mark.ma@emc.com>

### CODE REVIEW

**Comment by sharkconi** [comment]
1. Add test_ipmi_kcs.py function test
2. fix emu-bridge-helper path
3. fix bridge.conf path

@XiaowenJiang @xiar @fub2 @turtle-fly @JuneZhou19

**Comment by XiaowenJiang** [comment]
retest this please

File: test/functional/test_ipmi_kcs.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    - local sdr can work
+    - local user can work
+"""
+import unittest
+import os
+import yaml
+from infrasim import VM_DEFAULT_CONFIG
+from infrasim import qemu
+from infrasim import ipmi
+from infrasim import socat
```
Comment (turtle-fly) [inline test/functional/test_ipmi_kcs.py]: `ipmi` and `socat` are used in a limited test scenario and not in this test.

**Comment by XiaowenJiang** [comment]
retest this please

**Comment by XiaowenJiang** [comment]
retest this please

File: test/functional/test_ipmi_kcs.py (hunk 1 above)
Comment (sharkconi) [inline test/functional/test_ipmi_kcs.py]: Remove the unnecessory module and add user test case

**Comment by XiaowenJiang** [comment]
retest this please

**Comment by fub2** [comment]
:+1:

**Pull request merged**